generators/app/templates/src/containers/TicTacToe/GamePage/children/Game/game.component.js

```diff
@@ -0,0 +1,415 @@
+import React, { Fragment, useEffect, useState, useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useLiveUpdate, useNotification } from '@inrupt/solid-react-components';
+import moment from 'moment';
+import { ldflexHelper, errorToaster, buildPathFromWebId, notification } from '@utils';
+import ldflex from '@solid/query-ldflex';
+import { namedNode } from '@rdfjs/data-model';
+import tictactoeShape from '@contexts/tictactoe-shape.json';
+import Board from '../Board';
+import GameAccept from '../GameAccept';
+import { GameWrapper, Metadata } from './game.style';
+
+const possibleCombinations = [
+  [0, 4, 8],
+  [2, 4, 6],
+  [0, 1, 2],
+  [3, 4, 5],
+  [6, 7, 8],
+  [0, 3, 6],
+  [1, 4, 7],
+  [2, 5, 8]
+];
+type Props = { webId: String, gameURL: String };
+
+const Game = ({ webId, gameURL }: Props) => {
+  /** Game Logic */
+  const updates = useLiveUpdate();
+  const { timestamp } = updates;
+  const [gameDocument, setGameDocument] = useState(null);
+  const [gameData, setGameData] = useState({});
+  const [result, setResult] = useState(null);
+  const [isProcessing, setIsProcessing] = useState(false);
+  const inboxUrl = buildPathFromWebId(webId, process.env.REACT_APP_TICTAC_INBOX);
+  const { createNotification, createInbox } = useNotification(webId);
+  const [rival, setRival] = useState(null);
+  const { t } = useTranslation();
+
+  const sendNotification = useCallback(
+    async (player, content) => {
+      try {
+        /**
+         * Get full opponent game path
+         */
+        const gameSettings = buildPathFromWebId(
+          player,
+          `${process.env.REACT_APP_TICTAC_PATH}settings.ttl`
+        );
+
+        /**
+         * Find opponent inboxes from a document link
+         */
+        const inboxes = await notification.findUserInboxes([
+          { path: player, name: 'Global' },
+          { path: gameSettings, name: 'Game' }
+        ]);
+
+        /**
+         * If receiver doesn't has inbox will show an error and show users how
+         * to fix it.
+         */
+        if (inboxes.length === 0)
+          errorToaster(`${player} ${t('noInboxOpponent.message')}`, 'Error', {
+            label: t('noInboxOpponent.link.label'),
+            href: t('noInboxOpponent.link.href')
+          });
+        /**
+         * Find opponent game inbox if doesn't exist get global
```

Comment:
Should read:

"Find the opponent's game-specific inbox. If it doesn't exist, use the global inbox instead"